app/controllers/api/users_controller.rb

```diff
@@ -48,6 +47,19 @@ def delete_resource(type, id = nil, data = nil)
       super
     end
 
+    def set_current_group_resource(_type, id, data)
+      User.current_user.tap do |user|
+        raise "Can only edit authenticated user's current group" unless user.id == id
+        group_id = parse_id(data["current_group"], :groups)
```

Comment:
can you use parse_group(data["current_group"]) here ? it support both id/href as well as the identifying attr,i.e. be able to set current group by description.

Reply:
or maybe the parse_fetch_group we used earlier.

Reply:
@abellotti can't use `parse_fetch_group` because it uses `resource_search`. Will update to use `parse_group` though